### DISCUSSION TITLE
Unable to pass the verification

### DISCUSSION THREAD

**Discussion by SuzukiHonoka** [community/community#51742] [GitHub Education] [answered]
Hi there, I'm a university student from China. Recently I found my student pack has expired, So I go for re-verification, but all of my attemps are failed. I really need the help!

> I uploaded my university diploma, choosed the 'other' type, and the description are following:
> "This is my university diploma from Chinese higher-education information official website and also my student ID. I'm currently studying at QuanZhou normal university and major in computer science, the university doesn't provide its email system access to the students and any extra documents. I really love GitHub, please help!"

**Comment by joneslee0918** [comment]
https://github.com/orgs/community/discussions/51342
I think it may be helpful for you.

**Reply by SuzukiHonoka** [reply]
Thanks for the reply. 
Our university does not provide the email system access to the students. That's very sad.

**Reply by joneslee0918** [reply]
Getting access to your email system seems like a good solution.

**Comment by subeom7** [comment]
Did you use your school's email to verify? I was able to resolve this issue by using my school's email.

Also, make sure your diploma should display your expected graduation date. If it does not, ensure that you verify your enrollment using alternative school documents that confirm your graduation date.

Additionally, in the description, make sure to say you are open to contributing to open-source projects.

Lastly, link your institution's webpage: In the "Company" or "Website" section of your profile, consider adding a link to your educational institution's webpage to further validate your academic affiliation.

After making these updates, reapply for the GitHub Student Pack. If you continue to face difficulties, consider submitting the request through GitHub Support directly, as they can provide more specific guidance or investigate any issues with your application:

GitHub Education Support: [https://support.github.com/contact/education](https://support.github.com/contact/education)

Hope this helps!

**Reply by SuzukiHonoka** [reply]
Thanks for the reply.
Our university does not provide the email system access to the students.  And the diploma I provided is same as your proposed, I have no ideal why I still got rejected. That's very sad..

**Comment by samus-aran** [comment]
Hi @SuzukiHonoka 

Here are a few tips to help:

Ensure you select the most appropriate **evidence type** in the dropdown. The order of options indicates the preference of evidence with Academic Email being at the top. If you don't have an Academic email or Student ID, do try and acquire evidence that is positioned higher rather than lower in that option list.

If you have been repetitively unsuccessful using a particular evidence type (e.g. Student ID) we would recommend that you acquire and submit an alternative piece of evidence that can verify your student status.

If your original document is not in English then we advise you to take a photo of the original next to an English translation. This also include your profile information.

Ensure your evidence correspond to details in your GitHub profile. The verification process looks at your GitHub username, email and name ( as well as other points) to cross reference the evidence submitted. Do you use a variation of your name or a nickname?  It’s these small but important points to consider when applying that can make a much smoother process.

Let us know how you get on 👍

**Reply by SuzukiHonoka** [reply]
It worked, thanks 👍

**Comment by whitewatercn** [comment, marked as answer]
可以参考我的这个帖子再逝一逝 https://xlog.whitewater.wang/Github_Student_Developer_Pack
You can refer to this post to try again https://xlog.whitewater.wang/Github_Student_Developer_Pack

**Reply by SuzukiHonoka** [reply]
Very appreciated, it worked.
